tokio-stream/src/wrappers/broadcast.rs

```diff
@@ -0,0 +1,38 @@
+use crate::Stream;
+use async_stream::try_stream;
+use std::pin::Pin;
+use tokio::sync::broadcast::error::RecvError;
+use tokio::sync::broadcast::Receiver;
+
+use std::task::{Context, Poll};
+
+/// A wrapper around [`Receiver`] that implements [`Stream`]. Achieved by using the [`async-stream`] crate.
+///
+/// [`Receiver`]: struct@tokio::sync::broadcast::Receiver
+/// [`Stream`]: trait@crate::Stream
+/// [`async-stream`]: https://docs.rs/async-stream
```

Comment:
I don't think the documentation needs to say this.
```suggestion
/// A wrapper around [`Receiver`] that implements [`Stream`].
///
/// [`Receiver`]: struct@tokio::sync::broadcast::Receiver
/// [`Stream`]: trait@crate::Stream
```